### PR TITLE
Runbook for handling uptime alerts

### PR DESCRIPTION
**Pull request by wejdross**
## Checklist

- [ ] Try to isolate changes into separate PRs (to build a better changelog).
- [ ] Categorize the PR by setting a good title and adding one of the labels:
      `change`, `decision`, `requirement/quality`, `requirement/functional`, `dependency`
      as they show up in the changelog
- [ ] Link this PR to related issues if applicable.

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/appcat/GuaranteedUptimeTarget.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+It might be also worth checking for failing Kubernetes Objects and Composite:
+[source,bash]
+----
+#$instanceNamespace_generated_chars can be obtained in a way: `echo vshn-postgresql-my-super-prod-5jfjn | rev | cut -d'-' -f1 | rev` ===> 5jfjn
+kubectl --as cluster-admin get objects | egrep $instanceNamespace_generated_chars # here look for False objects and describe them to find out what is wrong
+kubectl --as cluster-admin get xvshn[TAB here for specific service] | egrep $instanceNamespace_generated_chars # also describe to read what happened
+----
+
+.Check logs of our comp-functions
```
Comment (Kidswiss) [inline docs/modules/ROOT/pages/how-tos/appcat/GuaranteedUptimeTarget.adoc:29]: I doubt this is helpful to anyone who's never worked with our AppCat.
Also the comp-functions don't necessarily say anything about why a specific service is not working.

File: docs/modules/ROOT/pages/how-tos/appcat/GuaranteedUptimeTarget.adoc [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+[source,bash]
+----
+kubectl -n syn-crossplane logs deployments/function-appcat-aeb2dbb03cf6 # <--- this number changes regularly
+----
+
+For stuck resources, You can create dummy label on object and then rollout restart crossplane function-appcat and provider-kubernetes.
```
Comment (Kidswiss) [inline docs/modules/ROOT/pages/how-tos/appcat/GuaranteedUptimeTarget.adoc:36]: Someone who has never used Crossplane might not know what a "stuck" resource is.

**Review by Kidswiss — CHANGES_REQUESTED** [review]

**Review by Kidswiss — APPROVED** [review]
Just one more nit, rest LGTM

File: docs/modules/ROOT/pages/how-tos/appcat/GuaranteedUptimeTarget.adoc [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+kubectl -n $instanceNamespace describe $failing_pod
+kubectl -n $instanceNamespace logs pods/$failing_pod
+----
+
+It might be also worth checking for failing Kubernetes Objects and Composite:
+[source,bash]
+----
+#$instanceNamespace_generated_chars can be obtained in a way: `echo vshn-postgresql-my-super-prod-5jfjn | rev | cut -d'-' -f1 | rev` ===> 5jfjn
+kubectl --as cluster-admin get objects | egrep $instanceNamespace_generated_chars # here look for False objects and describe them to find out what is wrong
+kubectl --as cluster-admin get xvshn[TAB here for specific service] | egrep $instanceNamespace_generated_chars # also describe to read what happened
```
[the remark this answers is not in the document]
Comment (wejdross) [reply docs/modules/ROOT/pages/how-tos/appcat/GuaranteedUptimeTarget.adoc:27]: I changed to that and also added a real life example, so it's going to be easier for engineers to go through our abstractions

[the remark this answers is not in the document]
Comment (Kidswiss) [reply docs/modules/ROOT/pages/how-tos/appcat/GuaranteedUptimeTarget.adoc:27]: I still shows as `kubectl --as cluster-admin get xvshn` for me not `kubectl --as cluster-admin describe xvshn`

**Pull request merged**